src/scales/scale.linearbase.js

```diff
@@ -205,7 +205,11 @@ export default class LinearScaleBase extends Scale {
       precision: tickOpts.precision,
       stepSize: tickOpts.stepSize
     };
-    const ticks = generateTicks(numericGeneratorOptions, me);
+    const {min, max} = me;
+    const ticks = generateTicks(numericGeneratorOptions, {
+      min: min - Math.abs(toDimension(opts.grace, min)),
```

Comment:
Question, should this be `% of min` or `% of max-min`. If `min === 0`, the a 5% grace would not change anything.

Reply:
True, but if applied to the range (`max - min`), and the value is number, say `10`, then `5` would be added each side, which would be unexpected(?)